src/screens/Settings/PrivacyAndSecuritySettings.tsx

```diff
@@ -71,6 +73,17 @@ export function PrivacyAndSecuritySettingsScreen({}: Props) {
               </SettingsList.BadgeText>
             )}
           </SettingsList.LinkItem>
+          <SettingsList.LinkItem
+            label={_(msg`Settings for activity alerts`)}
+            to={{screen: 'ActivityPrivacySettings'}}
+            contentContainerStyle={[a.align_start]}>
+            <SettingsList.ItemIcon icon={BellRingingIcon} />
+            <ItemTextWithSubtitle
+              titleText={<Trans>Allow others get notified of your posts</Trans>}
```

Comment:
```suggestion
              titleText={<Trans>Allow others to be notified of your posts</Trans>}
```
nit: As before, suggested wording tweak that reads slightly clearer imo